### PR TITLE
bug fixes and tweaks

### PR DESCRIPTION
**Pull request by hcarneiro**

### CODE REVIEW

File: js/build.js [hunk 1]
```diff
@@ -263,6 +280,8 @@ Fliplet.Widget.instance('chat', function (data) {
 
     getContacts(false).then(function () {
       return getConversations();
+    }).catch(function(error) {
```
Comment (hcarneiro) [inline js/build.js]: @squallstar This is what fixes that message. I was doing the catch in the wrong place. (Maybe it's still not entirely correct)

Comment (squallstar) [inline js/build.js]: @hcarneiro why is the catch triggered?

Comment (hcarneiro) [inline js/build.js]: @squallstar with that it isn't. I assume only if the `getConversations()` somehow fails, right?

Comment (squallstar) [inline js/build.js]: @hcarneiro can you please log the error with `console.warn(error)` ? otherwise we can't debug why it failed.

**Pull request merged**